package.json

```diff
@@ -12,7 +12,11 @@
     "test:cypress:response-validation": "cypress run --spec 'cypress/interceptors/response-validation.spec.js'",
     "test:cypress:response-modification": "cypress run --spec 'cypress/interceptors/response-modification.spec.js'",
     "test:cypress:request-validation": "cypress run --spec 'cypress/interceptors/request-validation.spec.js'",
-    "test:cypress:request-modification": "cypress run --spec 'cypress/interceptors/request-modification.spec.js'"
+    "test:cypress:request-modification": "cypress run --spec 'cypress/interceptors/request-modification.spec.js'",
+    "prettier:check": "npx prettier --check .",
```

Comment:
You can remove `npx` from all scripts, as we migrated to `yarn`, so no additional keywords are necessary.